test/python/test_cliffordsynthesis.py

```diff
@@ -115,6 +116,30 @@ def test_optimize_clifford_gates_at_minimal_two_qubit_gates(test_config: Configu
     print("\n", circ)
 
 
+@pytest.mark.parametrize("test_config", create_circuit_tests())
+def test_heuristic(test_config: Configuration) -> None:
+    """Test heuristic synthesis method."""
+    print(test_config.initial_circuit)
+    sys.stdout.flush()
+    circ, results = qmap.optimize_clifford(
```

Comment:
Both result and the other result below are unused and should be replaced by `_`

Reply:
Done.